src/Microsoft.DotNet.Build.Tasks/Targets/test-runtime-project.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Yeah, I have a comment about it in the change description.  There are a few things blocking me from doing so at the moment.